### DISCUSSION TITLE
Deploying snapshots to maven-central

### DISCUSSION THREAD

**Discussion by gstojsic** [jreleaser/jreleaser#1165] [Q&A] [answered]
Hi. I have been trying to familiarize myself with jreleaser. I want to use the "release to maven central" feature. I created this small test [project](https://github.com/gstojsic/jreleaser-test). At the moment I am trying to upload snapshots to maven central. When I run jreleaserFullRelease task I get the output in [out.txt](https://github.com/jreleaser/jreleaser/files/10418157/out.txt). so everything seems ok. However, when I try to use this dummy lib in another project it fails to build. This is the relevant build.gradle.kts snippet in the other project:
```
repositories {
    maven {
        setUrl("https://s01.oss.sonatype.org/content/repositories/snapshots/")
    }
}
dependencies {
    implementation("io.github.gstojsic.test:test-lib:1.0-SNAPSHOT")
    //...
}
```
The error I get is:
```
Execution failed for task ':compileKotlin'.
> Error while evaluating property 'filteredArgumentsMap' of task ':compileKotlin'.
   > Could not resolve all files for configuration ':compileClasspath'.
      > Could not find io.github.gstojsic.test:test-lib:1.0-SNAPSHOT.
        Searched in the following locations:
          - https://repo.maven.apache.org/maven2/io/github/gstojsic/test/test-lib/1.0-SNAPSHOT/maven-metadata.xml
          - https://repo.maven.apache.org/maven2/io/github/gstojsic/test/test-lib/1.0-SNAPSHOT/test-lib-1.0-SNAPSHOT.pom
          - https://s01.oss.sonatype.org/content/repositories/snapshots/io/github/gstojsic/test/test-lib/1.0-SNAPSHOT/maven-metadata.xml
          - https://s01.oss.sonatype.org/content/repositories/snapshots/io/github/gstojsic/test/test-lib/1.0-SNAPSHOT/test-lib-1.0-SNAPSHOT.pom
```
Upon closer inspection I noticed that maven-metadata.xml and friends haven't been uploaded to the maven central snapshots repo. Is there a switch I need to flip for that or a bug?
Thanks.

**Comment by aalmiray** [comment, marked as answer]
Hmm I can see that `maven-metadata.xml` is indeed missing from https://s01.oss.sonatype.org/content/repositories/snapshots/io/github/gstojsic/test/test-lib/1.0-SNAPSHOT/

This is indeed a bug. Filed https://github.com/jreleaser/jreleaser/issues/1166

**Reply by gstojsic** [reply]
Wow, that was quick!

**Reply by aalmiray** [reply]
Fixed. Binary bits should be ready in about 1hr when https://github.com/jreleaser/jreleaser/actions/runs/3920775828 finishes.
Take note that we do not post snapshot releases of plugins, only CLI.
